docs/STYLE_GUIDE.md

```diff
@@ -0,0 +1,151 @@
+# Formatting
```

Comment:
We are using rustfmt and clippy so info about that here could save a lot of reading :)